content/v2.0/monitor-alert/templates/AWS.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Monitor Amazon Web Services (AWS)
+seotitle: Monitor Amazon Web Services (AWS)
+description: >
+  Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
+menu:
+  v2_0:
+    parent: Monitor with templates
```

Comment:
- I don't know that "Monitor with templates" is the right name for the parents (as suggested above).
- Provide a short name for the left nav:

    ```yaml 
    menu:
      v2_0:
        parent: Monitor with templates
        name: AWS CloudWatch # <- This
```